transformers/run_eval.py

```diff
@@ -1,49 +1,152 @@
 import argparse
```

Comment:
This is the only file of interest for this draft PR. It should form a new PR on top of `main` once #20 is merged